authentication/using-tokens.mdx

```diff
@@ -66,23 +70,24 @@ import (
 )
 
 func main() {
-  req := http.NewRequest("GET", "https://flipt.your.instance/api/v1/flags", nil)
-  req.AddCookie(&http.Cookie{
-    Name:  "flipt_client_token",
-    Value: "gt6P_zIqTnCngfHDCpWb48ob5EBt3PqunUhpofNCNnc=",
-  })
-
-  resp, err := http.Do(req)
-  // ...
+req := http.NewRequest("GET", "https://flipt.your.instance/api/v1/flags", nil)
+req.AddCookie(&http.Cookie{
+Name: "flipt_client_token",
+Value: "gt6P_zIqTnCngfHDCpWb48ob5EBt3PqunUhpofNCNnc=",
+})
+
+resp, err := http.Do(req)
+// ...
 }
-```
+
+````
```

Comment:
It seems it has flattened all the Go code. Maybe this just works with our setup. But just made me double take.